index.html

```diff
@@ -1562,7 +1557,7 @@ <h3>Issuer</h3>
         </p>
 
         <p>
-A <a>verifiable credential</a> MUST have an <code>issuer</code> <a>property</a>.
+A <a>credential</a> MUST have an <code>issuer</code> <a>property</a>.
```

Comment:
```suggestion
A <a>credential</a> or <a>verifiable credential</a> MUST have an
<code>issuer</code> <a>property</a>.
```

Reply:
Rejected. A VC that is produced using JWT proofs does not have an issuer property as it has been replaced by the iss claim. This is the reason for specifically excluding verifiable credential in this text.

Reply:
I see.  I think that this JWT-specific lack should be noted explicitly.  Something like --
```suggestion
A <a>credential</a> or <a>verifiable credential</a> MUST have an
<code>issuer</code> <a>property</a> or equivalent, such as the
`iss` claim in a JWT-based <a>verifiable credential</a>.
```

Reply:
If you want to add verifiable credential then it should be "and" and not "or"

Reply:
No, it should be `<a>credential</a> or <a>verifiable credential</a>`, because any given instance will only be one or the other, because we've somehow tacitly agreed that `<a>credential</a>` means `<a>unverifiable credential</a>` (rather than `<a>credentials</a>` being the generic that includes all `<a>verifiable credentials</a>` and `<a>unverifiable credentials</a>`).

Even if that were not the case, `A <a>credential</a> and <a>verifiable credential</a> MUST ...` makes no sense.

Reply:
I think we will just have to agree to differ on this. Perhaps US and English grammar are different. But to me, "A or B must do something" means that only one of them must do it, whilst "Both A and B must do something" means that both must do it.

Reply:
I've edited my suggestion, changing the leading `A` to `Each` which I *think* makes the sentence mean the same in both UK and US English, even with the `or`, which I continue to think is appropriate here, because `each` document in question is one `or` the other, not one `and` the other.

Reply:
Yes this makes sense because each means we are talking about a singular object.